### ISSUE TITLE
Error using React-native 0.10.0-rc

### ISSUE THREAD

**Issue by maxs15** [Kureev/react-native-side-menu#90]
Hey guys,

I'm getting this error when using the last react native release 0.10.0-rc.
```
[error][tid:com.facebook.React.ShadowQueue][RCTModuleMethod.m:59] Argument 1 (RCTResponseSenderBlock) of RCTUIManager.configureNextLayoutAnimation must not be null
```
Any idea how we could fix this to support the last release ?

Thanks

**Comment by Kureev** [comment]
Have any ideas, @brentvatne ? :angel:

**Comment by exentrich** [comment]
0.11.0-rc also have this bug

**Comment by Kureev** [comment]
Should be solved in `0.12.1`

**Comment by maxs15** [comment]
Awesome, works perfectly :+1:
